account.go

```diff
@@ -82,6 +82,12 @@ func (c *Client) Accounts(ctx context.Context) ([]*Account, error) {
 	return c.parseBlob(bytes.NewReader(blob))
 }
 
+// FetchEncryptedAccounts fetches the user's encrypted accounts from LastPass.
+// The returned []byte can be parsed using the ParseEncryptedAccounts method.
+func (c *Client) FetchEncryptedAccounts(ctx context.Context) ([]byte, error) {
+	return c.fetchBlob(ctx)
+}
+
```

Comment:
I pushed a commit where `fetchBlob()` (renamed to `FetchEncryptedAccounts()`) is exported directly instead of adding another wrapper method. WDYT?

Reply:
I think that's fine. I added a wrapper method just in case you wanted the code to be separate for cosmetic reasons :)